src/App.js

```diff
@@ -60,7 +60,7 @@ function App() {
       <Routes>
         <Route
           path="/"
-          element={isLoggedIn ? <Home /> : <Navigate to="/signIn" />}
+          element={isLoggedIn ? <Home /> : <Navigate to="/home" />}
         />
```

Comment:
Works well; but we can shorten this even more. If we're going to render the Home page either way whether or not they are logged in, then it doesn't make sense to even do the check or to do a full page navigation. Instead we can simplify to:

```suggestion
          <Route path="/" element={<Home />} />
```

_or_ we can even use react-routers syntax for "optional segments" where we follow it by a question mark: https://reactrouter.com/en/main/route/route#:~:text=You%20can%20have%20optional%20static%20segments%2C%20too%3A

```suggestion
          <Route path="/home?" element={<Home />} />
```

^ with this we could remove line 65, since the above suggestion would match both for `"/"` _and_ for `"/home"`

Reply:
@akosasante thank you for showing new react options, I will commit it now